packages/dns-discovery/src/index.ts

```diff
@@ -58,7 +89,15 @@ export class PeerDiscoveryDns
     for await (const peer of this.nextPeer()) {
       if (!this._started) return;
       const peerInfos = multiaddrsToPeerInfo(peer.getFullMultiaddrs());
-      peerInfos.forEach((peerInfo) => {
+      peerInfos.forEach(async (peerInfo) => {
+        await this._components.peerStore.tagPeer(
+          peerInfo.id,
+          DEFAULT_BOOTSTRAP_TAG_NAME,
+          {
+            value: this._options.tagValue ?? DEFAULT_BOOTSTRAP_TAG_VALUE,
+            ttl: this._options.tagTTL ?? DEFAULT_BOOTSTRAP_TAG_TTL,
+          }
+        );
```

Comment:
What are these changes? If more modification to dns discovery is necessary then lets' do that and dog food that first.